oximeter/collector/src/lib.rs

```diff
@@ -53,6 +53,31 @@ pub use http_entrypoints::oximeter_api;
 pub use standalone::Server as StandaloneNexus;
 pub use standalone::standalone_nexus_api;
 
+#[usdt::provider(provider = "oximeter")]
+mod probes {
+    /// Fires when a producer is registered or updated.
+    fn producer__registered(
+        collector_id: &str,
+        producer_id: &str,
+        addr: &str,
+        interval: &str,
```

Comment:
Would this be more useful as an integer (maybe milliseconds?)?

Reply:
Done in 417c117fa